components/activity_and_insights/insights/top_threads/top_threads_table/top_threads_table.tsx

```diff
@@ -87,11 +93,52 @@ const TopThreadsTable = (props: Props) => {
         props.closeModal();
     }, [props.closeModal]);
 
-    const openThread = (post: Post) => {
-        trackEvent('insights', 'open_thread_from_top_threads_modal');
-        dispatch(selectPostAndParentChannel(post));
```

Comment:
I think this is the only use of `selectPostAndParentChannel`, right? If so we could delete it.

Reply:
Yeah we are using it further down on line 110, we just had to wrap it in a condition to determine if we wanted the modal or the RHS to open